entries.go

```diff
@@ -157,6 +157,52 @@ func GetAllEntries() ([]Item, error) {
 	return allEntries, err
 }
 
+// GetAllEntriesFiltered calls GetEntries with the given archive and starred parameters parameters, thus returning all articles as []wallabago.Item which match the filter
+func GetAllEntriesFiltered(archive, starred *bool) ([]Item, error) {
+	archiveInt := -1
+	if archive != nil {
+		switch *archive {
```

Comment:
such a switch should be handled by the application using this lib because the API want 0 or 1 and not true or false.